src/wallet/rpc/spend.cpp

```diff
@@ -1307,15 +1300,6 @@ RPCHelpMan sendall()
         },
         [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
         {
-            RPCTypeCheck(request.params, {
-                UniValue::VARR, // recipients
-                UniValue::VNUM, // conf_target
-                UniValue::VSTR, // estimate_mode
-                UniValueType(), // fee_rate, will be checked by AmountFromValue() in SetFeeEstimateMode()
-                UniValue::VOBJ, // options
-                }, true
-            );
-
```

Comment:
nit: could add the `get_array` at line 1316.
```c++
const UniValue& recipients{request.params[0].get_array()};
```

Reply:
Why? Also, seems unrelated?